src/update_engine/update_metadata.proto

```diff
@@ -85,47 +117,17 @@ message Signatures {
   repeated Signature signatures = 1;
 }
 
-message PartitionInfo {
+// BlobInfo is used to validate the source prior to the update or
+// the destination after the list of InstallOperations has run.
+message BlobInfo {
   optional uint64 size = 1;
   optional bytes hash = 2;
 }
 
 message DeltaArchiveManifest {
-  message InstallOperation {
-    enum Type {
-      REPLACE = 0;  // Replace destination extents w/ attached data
-      REPLACE_BZ = 1;  // Replace destination extents w/ attached bzipped data
-      MOVE = 2;  // Move source extents to destination extents
-      BSDIFF = 3;  // The data is a bsdiff binary diff
-    }
-    required Type type = 1;
-    // The offset into the delta file (after the protobuf)
-    // where the data (if any) is stored
-    optional uint32 data_offset = 2;
-    // The length of the data in the delta file
-    optional uint32 data_length = 3;
-
-    // Ordered list of extents that are read from (if any) and written to.
-    repeated Extent src_extents = 4;
-    // Byte length of src, not necessarily block aligned. It's only used for
-    // BSDIFF, because we need to pass that external program the number
-    // of bytes to read from the blocks we pass it.
-    optional uint64 src_length = 5;
-
-    repeated Extent dst_extents = 6;
-    // byte length of dst, not necessarily block aligned. It's only used for
-    // BSDIFF, because we need to fill in the rest of the last block
-    // that bsdiff writes with '\0' bytes.
-    optional uint64 dst_length = 7;
-
-    // Optional SHA 256 hash of the blob associated with this operation.
-    // This is used as a primary validation for http-based downloads and
-    // as a defense-in-depth validation for https-based downloads. If
-    // the operation doesn't refer to any blob, this field will have
-    // zero bytes.
-    optional bytes data_sha256_hash = 8;
-  }
-  repeated InstallOperation install_operations = 1;
+  // The update procedure for the main partition (USR-A or USR-B). Once
+  // complete it should match the hash specified in new_partition_info.
+  repeated InstallOperation partition_operations = 1;
```

Comment:
That list of install operations is only for the partition, so I renamed it from being overly generic to specify that. Previously there was kernel_install_operations as a second list. The type doesn't change since a new install operation sequence will be added in the future, reusing the same type.